memcached-operator/deploy/olm-catalog/memcached-operator/0.5.0/memcached-operator.v0.5.0.clusterserviceversion.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
     alm-examples: '[{"apiVersion":"cache.example.com/v1alpha1","kind":"Memcached","metadata":{"name":"example-memcached"},"spec":{"size":3}}]'
     capabilities: Basic Install
   name: memcached-operator.v0.5.0
-  namespace: placeholder
+  namespace: default
```

Comment:
The namespace will be overwritten by OLM when installed using a catalogsource and subscription, which in my eyes is the only way of installing with OLM. I have no objections with using `default` as a placeholder for CSVs, but the plus side of using `placeholder` is that it discourages using `kubectl apply -f` to add the CSV directly to a cluster

Reply:
@estroz After second thoughts we'll most likely generate the CSVs as part of the getting-started guide, and we won't be using this CSV directly. Plus the fact that we can't use this CSV right now without a valid image.
https://github.com/operator-framework/getting-started/pull/36#discussion_r277429727

So I would suggest we close this PR and keep `namespace: placeholder` for now.